### DISCUSSION TITLE
如何修改Node中字体的fontFamily

### DISCUSSION THREAD

**Discussion by HongTaoSix** [antvis/X6#3845] [Q&A]
![QQ截图20230812142959](https://github.com/antvis/X6/assets/32625064/1962c961-d26c-44c4-be77-71ada4c03baf)
如图，我通过attr方法设置text/fontFamily，发现是无效的，同样的方式设置fontSize、fontWeight等都是实时变化的，请问是我使用的版本有问题，还是寻找的属性路径不对？

**Comment by NewByVector** [comment]
可以通过查看 dom 元素看下改属性是否生效。
